app/views/templates/script/package_action.erb

```diff
@@ -97,18 +109,27 @@ handle_zypp_res_codes () {
   <%-
     action = 'install' if action == 'group install'
     action = 'remove' if action == 'group remove'
+    is_pkg_upgrade = false
     if action == 'group update' || action == 'update'
       if input('package').blank?
         action = 'upgrade'
       else
         action = '--only-upgrade install'
+        is_pkg_upgrade = true
       end
     end
   -%>
   [ -x "$(command -v subscription-manager)" ] && subscription-manager refresh
   export DEBIAN_FRONTEND=noninteractive
   apt-get -y update
-  apt-get -o APT::Get::Upgrade-Allow-New="true" -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y <%= input("options") %> <%= action %> <%= input("package") %>
+
+  PACKAGES="<%= input("package") %>"
+  <% if is_pkg_upgrade -%>
+    PACKAGES="$(get_upgradeable_packages "<%= input("package") %>")"
+    echo "Only try to upgrade packages which are already installed on the system: ${PACKAGES}"
+  <% end -%>
```

Comment:
```suggestion
  <% if is_pkg_upgrade -%>
    PACKAGES="$(get_upgradeable_packages "<%= input("package") %>")"
    echo "Only try to upgrade packages which are already installed on the system: ${PACKAGES}"
  <% else -%>
    PACKAGES="<%= input("package") %>"
  <% end -%>
```

Or something similar, so that `PACKAGES=…` only shows up once in the rendered version? Otherwise, it seems confusing to the (possible) reader